### ISSUE TITLE
WCAG 4.1.2 test 22: false positives?

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#221]
[Test document 16](https://github.com/veraPDF/veraPDF-wcag-algs/files/9110711/test-document-16.pdf) fails on WCAG 4.1.2 test 22 (invalid list element). However, it appears to produce this error on well-formed lists throughout the document.

Can you check whether these are false positives?
